### ISSUE TITLE
[AVM Question/Feedback]: Where best to raise issues when they also relate to the provider or one of the libraries?

### ISSUE THREAD

**Issue by JWilkinsonMB** [Azure/terraform-azurerm-avm-ptn-alz#154]
### Check for previous/existing GitHub issues

- [x] I have checked for previous/existing GitHub issues

### Description

This module has a dependency on [terraform-provider-alz](https://github.com/Azure/terraform-provider-alz) which is in turn dependent on [Azure-Landing-Zones-Library](https://github.com/Azure/Azure-Landing-Zones-Library) & [alzlib](https://github.com/Azure/alzlib).

As this is the only repository that's directly under the governance of AVM, is there a preference to use this repo as a centralized hub for issues, or should they also / instead be raised upstream?

E.g. a bug could be raised:
- Only in this repo, and allow the maintainers to update upstream
- Only in the upstream repo that is causing the bug (if the reporter knows where the issue lies, otherwise raise here and a maintainer will move it)
- In both this repo and in upstream

A feature request could be raised:
- Only in this repo, and allow the maintainers to update upstream
- In the first upstream repo that would implement the feature, and allow maintainers to propagate downstream 
- In every repo that would need to be updated to implement the feature

Whatever the preference is, could the issue templates in the various repositories be updated to reflect this?

**Comment by matt-FFFFFF** [comment]
Hi @JWilkinsonMB 

If you aren't sure then use this repo for issues and features. They may be moved once triaged but the important thing is that they are logged in the first place.

Does that work? #RR

**Comment by JWilkinsonMB** [comment]
So as an example, I raised the feature request [alzlib #191](https://github.com/Azure/alzlib/issues/191) to make it possible to customise the suffix that gets appended to a role definition, but ultimately I'd like it implemented here so I can actually use it, which will also require support in the provider. 

I created the issue in alzlib because that's the first piece of code that would need updating, which is typically the right thing to do for OSS projects. 

It might be different in this case given all 4 repos are relatively tightly coupled, and share the same maintainers. 

So for this feature request, would the preference be for me to raise it in all 3 repos that will need updating (module, provider & library), to raise it in this repo, as the ultimate destination for the enhancement and where there is a more robust triage process, or to raise it only in upstream?

I'm good with any approach, just want to minimise the overhead for you so I can align the way I raise issues with the way you want to work on them.
